great_expectations/render/components.py

```diff
@@ -627,8 +630,8 @@ def __init__(
         header: Optional[RenderedAtomicValue] = None,
         template: Optional[str] = None,
         params: Optional[dict] = None,
-        header_row: Optional[List[RenderedAtomicValue]] = None,
-        table: Optional[List[List[RenderedAtomicValue]]] = None,
+        header_row: Optional[List[RendererTableValue]] = None,
+        table: Optional[List[List[RendererTableValue]]] = None,
```

Comment:
Would it make things easier if these could never be `None` but could be empty lists?